README.md

```diff
@@ -343,6 +346,56 @@ You can find your dayjs locale here:
 
 https://github.com/iamkun/dayjs/tree/dev/src/locale
 
+## Enriched Events Logic (Performance Boost)
+
+### Performant Data Structure
+
+We create and cache a dictionary keyed by date with all the events properties plus the overlapping counting:
+
+```
+{
+ [YYYY-MM-DD]: {
+    ...all event information,
```

Comment:
Should this be an array of events instead ?

Reply:
Good catch, is an array now